seccomp/mount.go

```diff
@@ -604,6 +703,24 @@ func (m *mountSyscallInfo) createBindMountPayload(
 }
 
 func (m *mountSyscallInfo) String() string {
-	return fmt.Sprintf("source: %s, target = %s, fstype = %s, flags = %#x, data = %s",
-		m.Source, m.Target, m.FsType, m.Flags, m.Data)
+	return fmt.Sprintf("source: %s, target = %s, fstype = %s, flags = %#x, data = %s, root = %s, cwd = %s",
+		m.Source, m.Target, m.FsType, m.Flags, m.Data, m.root, m.cwd)
+}
+
+// Method addresses scenarios where the process generating the mount syscall has
+// a 'root' attribute different than default one ("/"). This is typically the case
+// in 'chroot'ed environments. Method's goal is to make all the require adjustments
+// so that sysbox-fs can carry out the mount in the expected context.
+func (m *mountSyscallInfo) rootAdjust() error {
+
+	root := m.syscallCtx.root
+
+	if root == "/" {
+		return nil
+	}
+
+	// 'Target' attribute will always require adjustment in chroot'ed envs.
```

Comment:
How about the source attribute?

Reply:
Excluding bind-mount operations, can you think about a scenario that would require "source" attribute to hold a FS path in the operations carried over by sysbox-fs? I'm asking coz prepending a path is not trivial in the "source" case, as this field is somewhat overloaded (FS types, FS paths, etc).

See my previous paragraph about how kernel is obviating "root" attribute in chroot'ed contexts -- that's why i'm obviating the bind-mount case.

Reply:
As mentioned above, sysbox-fs needs to deal with bind-mounts inside chroot jails, so yes, we need to take "source" attribute in consideration too.